### PR TITLE
vulnerability-management

### PR DESCRIPTION
**Pull request by Quadrisheriff**
The article is not complete, you haven't added the devices, patches, and request sections yet. However, I have added the required screenshots and also edited some parts of the article that are available now.

### CODE REVIEW

File: _docs/vulnerabilitymanagement.md [hunk 1]
```diff
@@ -5,176 +5,141 @@ tag: UI
 sidebar:
   nav: "sidebar-sample"
 ---
-The **Vulnerability Management** module enables you to view your XM Cyber
-sensors in a layout specifically designed to guide you in continuous
-vulnerability assessments and remediation for your Windows devices.
 
-This module displays information about vulnerabilities affecting your
-system from multiple perspectives:
+The **Vulnerability Management** module displays information about the vulnerabilities affecting your system from multiple perspectives. This module allows you to view your XM Cyber sensors in a layout specifically designed to guide you in continuous vulnerability assessments and remediation for your Windows devices.
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:9]:
```suggestion
The **Vulnerability Management** module displays information about the vulnerabilities affecting your system. Use this module to view data about your network, to review the vulnerability assessments resulting from the scenarios you're running and to understand how to remediate and mitigate these vulneabilities for your Windows devices.
```

@Quadrisheriff have you read the guidelines in the style guide I shared with you?

File: _docs/vulnerabilitymanagement.md [hunk 2]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
-    those patches including which vulnerabilities each fixes
+- The devices affected by vulnerabilities, and information about all of the vulnerabilities affecting each of them.
 
-The overview and details provided from this module help you prioritize
-vulnerabilities so you can easily focus on and fix the most urgent
-vulnerabilities and the most vulnerable devices, with the most
-comprehensive patches. Once you've patched the vulnerabilities, you can
-also send mails directly from the system notifying of actions taken
-to any parties who submitted related requests.
+- The patches that will remediate vulnerabilities, and details about those patches including which vulnerabilities each fixes.
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:17]:
```suggestion
- The patches that remediate these vulnerabilities, and details about those patches including which vulnerabilities each of them fixes.
```

File: _docs/vulnerabilitymanagement.md [hunk 3]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
+
+The overview and details provided by this module help you to prioritize vulnerabilities so you can easily focus on and fix the most urgent vulnerabilities and the most vulnerable devices with the most comprehensive patches.
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:19]:
```suggestion
The overview and details provided by this module help you to prioritize vulnerabilities so you can easily focus on and fix the most urgent vulnerabilities and the most vulnerable devices by using the most comprehensive patches. 
```

File: _docs/vulnerabilitymanagement.md [hunk 4]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
+
+The overview and details provided by this module help you to prioritize vulnerabilities so you can easily focus on and fix the most urgent vulnerabilities and the most vulnerable devices with the most comprehensive patches. 
+
+Once you've patched the vulnerabilities, you can also send mails directly from the system notifying of actions taken to any parties who submitted related requests.
 
 Navigate to the Vulnerabilities Management module
 ------------------------------------------------
 
-From the side panel, click ![icvulnerability](../../assets/images/icvulnerability.png) to navigate to **Vulnerabilities Management**:
+To navigate to the **Vulnerabilities Management** module, click ![icvulnerability](../../assets/images/icvulnerability.png) from the navigation panel. The default **Overview** tab is displayed with an overview of the vulnerabilities in your system.
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:26]:
```suggestion
Click ![icvulnerability](../../assets/images/icvulnerability.png) from the navigation panel to navigate to the **Vulnerabilities Management** module. The default **Overview** tab is displayed with an overview of the vulnerabilities in your system similar to the following image:
```

File: _docs/vulnerabilitymanagement.md [hunk 5]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
 
--   Requests
+- [Requests](#requests)
 
 Overview
 --------
 
-The Overview tab displays an overview of the current status of your
-scenarios, including a summary of:
+<img src="..\..\assets\images\vulnerabilitiesoverview.png" alt="vulnerability overview" style="zoom:100%;" />
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:45]: delete this

File: _docs/vulnerabilitymanagement.md [hunk 6]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
 
--   The most recently discovered vulnerabilities (based on the
-    information in the Vulnerabilities tab)
+The Overview tab displays an overview of the current status of your scenarios, including a summary of:
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:47]:
```suggestion
The Overview tab appears similar to the following: 
<img src="..\..\assets\images\vulnerabilitiesoverview.png" alt="vulnerability overview" style="zoom:100%;" />

This tab displays an overview of the current status of your scenarios, including a summary of:
```

File: _docs/vulnerabilitymanagement.md [hunk 7]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
 
--   The vulnerabilities not yet handled, divided by severity, and how
-    many vulnerabilities have been in the network for the indicated
-    periods of time (in days) (based on the information in the
-    Vulnerabilities tab)
+- The most recently discovered vulnerabilities (based on the information in the Vulnerabilities tab).
 
--   The most recent pending patch requests (based on the information in
-    the Requests tab)
+- The vulnerabilities not yet handled - divided by severity, and how many vulnerabilities have been in the network for the indicated periods of time (in days) (based on the information in the Vulnerabilities tab).
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:51]:
```suggestion
- Vulnerabilities not yet handled - divided by severity, and how many vulnerabilities have been in the network for the indicated periods of time (in days) (based on the information in the Vulnerabilities tab).
```

File: _docs/vulnerabilitymanagement.md [hunk 8]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
-    displays the number of patches that need to be applied
+- The most recent pending patch requests (based on the information in the Requests tab).
+
+- Missing patches (based on the information in the Patches tab) - displays the number of patches that need to be applied.
 
 ### Gaining insight
 
--   Hover over any of the pies or charts to view additional details
-    about the summary information displayed.
+To gain an insight on a certain summary or status:
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:59]:
```suggestion
To gain insights on a certain summary or status:
```

File: _docs/vulnerabilitymanagement.md [hunk 9]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
----------------
+- Click **View all** for any of the summaries. XM Cyber redirects you to the relevant tab.
 
-The Vulnerabilities tab displays all of the vulnerabilities that were
-found in your network in any of your scenarios, organized per vulnerability. The table appears similar to the following example:
+Vulnerabilities
+----------------
 
-### Manipulating the data
+The Vulnerabilities tab displays all of the vulnerabilities that were found in your network in any of your scenarios, organized per vulnerability. The table appears similar to the following example:
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:68]:
```suggestion
The Vulnerabilities tab displays all of the vulnerabilities that were found in your network based on any of the campaigns that have run. The list is organized per vulnerability. The table appears similar to the following example:
```

File: _docs/vulnerabilitymanagement.md [hunk 10]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
-    -   First Detected
+  -  Vulnerabilities
+  -  CVSS v3
+  -  Affected Devices
+  -  Products
+  -  First Detected
+  -  Updated
 
-    -   Updated
+To sort and filer your vulnerabilities data:
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:91]:
```suggestion
To sort and filer vulnerability data:
```

File: _docs/vulnerabilitymanagement.md [hunk 11]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
 
--   Click any column to sort (up or down) the entire table by that
-    column.
+-   Click any column. Press <img src="..\..\assets\images\sortup.png" alt="sort up" style="zoom:60%;" /> or <img src="..\..\assets\images\sortdown.png" alt="sort down" style="zoom:60%;" /> to sort the entire table by that column.
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:93]:
```suggestion
1. Click any column. 
2. Click <img src="..\..\assets\images\sortup.png" alt="sort up" style="zoom:60%;" /> or <img src="..\..\assets\images\sortdown.png" alt="sort down" style="zoom:60%;" /> to sort the entire table by that column.
```

File: _docs/vulnerabilitymanagement.md [hunk 12]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
 
--   Filter data:
+-   Type any keyword in the search box to filter the table and view only rows containing that keyword.
 
-    -   Type any keyword in the search box to filter the table and view
-        only rows containing that keyword.
+-   Click <img src="..\..\assets\images\star.png" alt="star" style="zoom:60%;" /> to
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:97]: @Quadrisheriff something's missing here.

File: _docs/vulnerabilitymanagement.md [hunk 13]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
 
-    -   Type any keyword in the search box to filter the table and view
-        only rows containing that keyword.
+-   Click <img src="..\..\assets\images\star.png" alt="star" style="zoom:60%;" /> to 
 
-    -   Click to
+-   Click <img src="..\..\assets\images\casesensitive.png" alt="case sensitive" style="zoom:60%;" /> to toggle between case sensitive, and not case sensitive.
 
-    -   Click to toggle between case sensitive, and not case sensitive.
+**Note -- The data of hidden columns are still displayed during filtering**
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:101]:
```suggestion
**Note -- Data in hidden columns is displayed even when filtered.**
```

File: _docs/vulnerabilitymanagement.md [hunk 14]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
 
--   Click to export to a CSV (comma separated values) file:
+To download your vulnerabilities data:
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:103]:
```suggestion
To download vulnerability data:
```

File: _docs/vulnerabilitymanagement.md [hunk 15]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
 
-    -   the data for the selected rows, when one or more is selected
+- Click **Export** to export your data to a CSV (comma separated values) file. 
 
-    -   the data for all of the rows in the system, when no rows are
-        selected
+- The data for the selected rows will be exported when one or more rows are selected.
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:107]:
```suggestion
- Data is exported when at least one row is selected.
```

File: _docs/vulnerabilitymanagement.md [hunk 16]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
 
-**Note:** Even when you've hidden a column, its data is still used when
-filtering.
+- The data for all of the rows in the system will be exported when no rows are selected.
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:109]:
```suggestion
- When no rows are selected, all of the data the system is exported.
```

File: _docs/vulnerabilitymanagement.md [hunk 17]
```diff
@@ -5,176 +5,141 @@ tag: UI
...
 
-As you select rows, the summary at the bottom of the table updates with
-the total number of unique devices affected and the total number of
-unique patches that can fix those vulnerabilities.
+As you select rows, the summary at the bottom of the table updates with the total number of unique devices affected and the total number of unique patches that can fix those vulnerabilities.
 
-Often times, the vulnerabilities are fixed with shared patches, as in
-the above example. This makes the overall picture clearer, and easier to
-prioritize which vulnerabilities should be handled first.
+Often times, the vulnerabilities are fixed with shared patches, as in the above example. This makes the overall picture clearer, and easier to prioritize which vulnerabilities should be handled first.
```
Comment (spinningrachel) [inline _docs/vulnerabilitymanagement.md:120]:
```suggestion
Often times, the vulnerabilities are fixed with shared patches, as in the above example. This makes the overall picture clearer. In this way, it is then easier to prioritize which vulnerabilities that should be handled first.
```

**Review by spinningrachel — CHANGES_REQUESTED** [review]

File: _docs/vulnerabilitymanagement.md (hunk 12 above)
Comment (Quadrisheriff) [reply _docs/vulnerabilitymanagement.md:98]: The description was not in the original article

**Review by spinningrachel — APPROVED** [review]

**Pull request merged**